params/cspc_lidar.yaml

```diff
@@ -1,10 +1,10 @@
 cspc_lidar:
   ros__parameters:
-    port: /dev/sc_mini
-    frame_id: laser_link
+    port: /dev/ttyUSB0
```

Comment:
Please return back /dev/sc_mini
The idea is that whe should setup udev rule that will create symbol link /dev/sc_mini. It is needed to avoid problems when a few USB tty devices are connected.
Here is udev rules: https://github.com/KPI-Rover/ros_cspc_lidar_coin_d4a/blob/main/sc_mini.rules
Here is instruction: https://github.com/KPI-Rover/ros_cspc_lidar_coin_d4a

Please do not forget add this as dedicated step for RPI preparation manual.